lib/stack_master/commands/helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/concern'
+
+module StackMaster
+  module Commands
+    module Helper
+      extend ActiveSupport::Concern
+
+      included do
```

Comment:
I'm not sure the use of active support concern is necessary here. Could we just use regular ruby to avoid depending on AS?

```
module Helper
  def window_size
    ...
  end
end
```